.github/workflows/base.yaml

```diff
@@ -103,6 +99,8 @@ jobs:
         run: make check
       - name: check-license-headers
         run: make check-license-headers
+      - name: tidy
+        run: make tidy
```

Comment:
We'll need to run `make tidy` before `make check`, so that the check is performed after dependencies are downloaded, giving an accurate picture about static code vulnerabilities.